### ISSUE TITLE
Server immediately closes after client connects

### ISSUE THREAD

**Issue by woojoo666** [kaytat/SimpleProtocolServer#3]
I probably missed a step somewhere but what I did was

* download simpleserver.exe (0.0.0.4)
* double-check the sampling rate of my Windows audio device
* run it
* server shows message `SimpleTcpServer::waitForClient ...`
* install SimpleProtocolPlayer NG from F-Droid (version 2019.10.0)
* open Simple Protocol Player on my android device
* connect to port `12345` and use the same sampling rate that I checked earlier
* once I click the play button, there is a message "Unable to stream"
* and on the server side, it says

```
Peer addr: 192.168.0.XXX:XXXXX
Press Enter to quit...
Stopping capture...
Received stop event after 0 passes and 0 frames
```
* (note that I redacted my the ip and port), but the rest is the same
* so it seems like the server is stopping once the client connects

* I wasn't sure if I should install Pulse Audio as well, so I first tried SimpleProtocolServer without Pulse Audio, and it didn't work
* and then I tried installing Pulse Audio 15.0 from [here](https://pgaskin.net/pulseaudio-win32/) and tried running the server again, same issue
* would love some help or guidance!

**Comment by kaytat** [comment]
"Stopping capture..." gets printed when there's input in the keyboard. Are you pressing enter after starting the .exe? You shouldn't have to.

**Comment by woojoo666** [comment]
Sorry just saw this response. So I went to try again today, and it seems to be working fine now. I'm pretty sure I wasn't pressing a key on the keyboard before, and I'm using the exact same devices, the exact same executable, and exact same android app as I was before, so perhaps it was just user error after all. Sometimes if there's nothing playing on the PC side and I start playing something, I have to press "Play" in the android app again, but that's no issue at all. I've learned to just start my music before I connect from the android side.

The latency of this program is the lowest of all the programs I've tried (I was using SWYH before which had like 8 seconds latency), so this is amazing and I'll definitely be using this from now on. Thank you!!
